api/decompress_chunk.md

```diff
@@ -10,8 +10,8 @@ api:
 
 # decompress_chunk() <Tag type="community">Community</Tag>
 
-If you need to modify or add data to a chunk that has already been
-compressed, you need to decompress the chunk first. This is especially
+If you need to modify or add data a lot of data to a chunk that has already been
+compressed, you should decompress the chunk first. This is especially
```

Comment:
```suggestion
If you need to modify or add a lot of data to a chunk that has already been
compressed, you should decompress the chunk first. This is especially
```

Reply:
insert without unique constraints is totally fine, but maybe thats too nuanced